### PR TITLE
SAR-9167 [MEI] - Added new FS and stub for registration

### PR DESCRIPTION
**Pull request by refaque-ahmad**

### CODE REVIEW

File: app/uk/gov/hmrc/minorentityidentification/config/AppConfig.scala [hunk 1]
```diff
@@ -30,4 +30,13 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig) extends FeatureSwitchi
   val graphiteHost: String = servicesConfig.getString("microservice.metrics.graphite.host")
 
   val timeToLiveSeconds: Long = servicesConfig.getInt("mongodb.timeToLiveSeconds").toLong
+
+  lazy val desStubBaseUrl: String = servicesConfig.getString("microservice.services.des.stub-url")
```
Comment (woollett-r) [inline app/uk/gov/hmrc/minorentityidentification/config/AppConfig.scala:34]: These two base urls will be read from the application.conf file. However, the DES service is not defined there. Checkout the sole trader or incorporated entity backends.

File: app/uk/gov/hmrc/minorentityidentification/featureswitch/core/config/FeatureSwitchingModule.scala [hunk 2]
```diff
@@ -33,6 +33,11 @@ class FeatureSwitchingModule extends Module with FeatureSwitchRegistry {
   }
 }
 
+case object DesStub extends FeatureSwitch {
+  override val configName: String = "feature-switch.registration-stub"
```
Comment (woollett-r) [inline app/uk/gov/hmrc/minorentityidentification/featureswitch/core/config/FeatureSwitchingModule.scala]: The feature switch could be used to stub any call to DES so a slightly more generic name might be advisable. Again take a look at the other backend services.

File: app/uk/gov/hmrc/minorentityidentification/config/AppConfig.scala (hunk 1 above)
Comment (refaque-ahmad) [reply app/uk/gov/hmrc/minorentityidentification/config/AppConfig.scala:34]: good spot! forgot add before but now added on application.conf

File: app/uk/gov/hmrc/minorentityidentification/featureswitch/core/config/FeatureSwitchingModule.scala (hunk 2 above)
Comment (refaque-ahmad) [reply app/uk/gov/hmrc/minorentityidentification/featureswitch/core/config/FeatureSwitchingModule.scala]: updated

**Review by woollett-r — APPROVED** [review]

**Pull request merged**